.github/actions/auto-major-minor-release-v1/action.yml

```diff
@@ -140,7 +140,7 @@ runs:
         # Required for the GH CLI
         GH_TOKEN: ${{ inputs.project_token }}
     - name: Slack Notification
-      uses: rtCamp/action-slack-notify@b24d75fe0e728a4bf9fc42ee217caa686d141ee8 # tag=v2
+      uses: rtCamp/action-slack-notify@e31e87e03dd19038e411e38ae27cbad084a90661 # tag=v2
```

Comment:
[A bunch of bug fixes](https://github.com/rtCamp/action-slack-notify/releases), no functional changes that should affect our usage.